test/expect_result.jl

```diff
@@ -0,0 +1,81 @@
+using Arrow
+using Test
+
+enumerate_row(value) = enumerate(value)
+enumerate_row(value::String) = [(1, value)]
+
+function generate_arrow(results)
+    arrowed_results = Dict()
+    for result in results
+        key = RAITest.relation_id(result[1], result[2])
```

Comment:
might be worth pulling out `result[2]` to a variable since it's used quite a bit and is kind of opaque as to what it represents

Reply:
Good point. Done.